lib/api/core/janitor.js

```diff
@@ -362,6 +465,109 @@ class Janitor {
     process.exit(0);
   }
 
+  _loadJson (filePath) {
+    return new Bluebird((resolve, reject) => {
+      fs.readFile(filePath, 'utf8', (error, rawData) => {
+        if (error) {
+          return reject(error);
+        }
+
+        try {
+          const data = JSON.parse(rawData);
+
+          resolve(data);
+        }
+        catch (e) {
+          reject(e);
+        }
+      });
+    });
+  }
+
+  _createRoles (roles) {
+    if (! roles) {
+      return Bluebird.resolve();
+    }
+
+    const promises = Object.keys(roles).map(roleId => {
```

Comment:
(nitpicking) This can be simplified using `Bluebird.map`:

```js
return Bluebird.map(Object.keys(roles), roleId => {
  const request = new Request({
    controller: 'security',
    action: 'createOrReplaceRole',
    refresh: 'wait_for',
    _id: roleId,
    body: roles[roleId]
  });
  return this.kuzzle.funnel.processRequest(request);
});
```

If you approve of this proposal, then it should also be applied to other _createXxx functions